src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -777,198 +1120,238 @@ s32 func_80A12B78(EnGo* this, PlayState* play) {
     return 0;
 }
 
-s32 func_80A12C48(EnGo* this, PlayState* play, s32 arg2) {
+/**
+ * Change the Goron's animation.
+ *
+ * Goron animations come from one of three categories
+ * - Basic Goron animations
+ * - Goron Gymnastics like Stretches, Cheers, etc... (Taisou)
+ * - Spring has Arrived Cutscene Animationstodo (Hakugin)
+ * Changing animations with this function handles the transitions between the three categories.
+ *
+ * @return true if animation request was valid
+ */
+s32 EnGo_ChangeAnimation(EnGo* this, PlayState* play, EnGoAnimationIndex anim) {
```

Comment:
```suggestion
s32 EnGo_ChangeAnim(EnGo* this, PlayState* play, EnGoAnimationIndex animIndex) {
```
We've just recently started to standardize these types of functions.